intel-sgx/dcap-retrieve-pckid/src/lib.rs

```diff
@@ -34,7 +35,25 @@ impl<'a> fmt::Display for PrintHex<'a> {
     }
 }
 
-pub fn retrieve_pckid_str() -> Result<String, &'static str> {
+pub struct PckCertId {
```

Comment:
Could you rename this `PckId`? (It identifies a platform certification key, not a certificate, see https://download.01.org/intel-sgx/latest/dcap-latest/linux/docs/Intel_SGX_ECDSA_QuoteLibReference_DCAP_API.pdf table 9)

Reply:
Got it.
Also related to comment above. I think the struct `PckId` is still needed, since it need a struct to wrap the ppid and qe_id instead of using a tuple here.

Reply:
Renamed